### PR TITLE
Add some annotation of instruction opcodes.

### PR DESCRIPTION
**Pull request by Calring**
Thank you!

### CODE REVIEW

**Comment by lzshhxx** [comment]
bot test this please

File: gas/config/tc-loongarch.c [hunk 1]
```diff
@@ -667,18 +670,20 @@ check_this_insn_before_appending (struct loongarch_cl_insn *ip)
...
     {
       /* For bstr(ins|pick).[wd] */
       if (ip->args[2] < ip->args[3])
         as_fatal (_ ("bstr(ins|pick).[wd] require msbd >= lsbd"));
     }
-  else if (ip->insn->mask != 0 && (ip->insn_bin & 0xfe0003c0) == 0x04000000 &&
-           (strcmp ("csrxchg", ip->name) == 0 ||
-            strcmp ("gcsrxchg", ip->name) == 0))
-    as_fatal (_ ("g?csrxchg require rj != $r0 && rj != $r1"));
+  else if (ip->insn->mask != 0 && (ip->insn_bin & 0xfe0003c0) == 0x04000000 &&
```
Comment (yetist) [inline gas/config/tc-loongarch.c:683]: Trailing space

File: gas/config/tc-loongarch.c [hunk 2]
```diff
@@ -667,18 +670,20 @@ check_this_insn_before_appending (struct loongarch_cl_insn *ip)
...
       /* For bstr(ins|pick).[wd] */
       if (ip->args[2] < ip->args[3])
         as_fatal (_ ("bstr(ins|pick).[wd] require msbd >= lsbd"));
     }
-  else if (ip->insn->mask != 0 && (ip->insn_bin & 0xfe0003c0) == 0x04000000 &&
-           (strcmp ("csrxchg", ip->name) == 0 ||
-            strcmp ("gcsrxchg", ip->name) == 0))
-    as_fatal (_ ("g?csrxchg require rj != $r0 && rj != $r1"));
+  else if (ip->insn->mask != 0 && (ip->insn_bin & 0xfe0003c0) == 0x04000000 && 
+        /* csrxchg  rd, rj, csr_num  */
```
Comment (yetist) [inline gas/config/tc-loongarch.c:684]: Indent with 2 spaces

File: gas/config/tc-loongarch.c [hunk 3]
```diff
@@ -656,9 +656,12 @@ check_this_insn_before_appending (struct loongarch_cl_insn *ip)
       ip->reloc_num++;
     }
   else if (ip->insn->mask == 0xffff8000 &&
-           ((ip->insn_bin & 0xfff00000) == 0x38600000 ||
-            (ip->insn_bin & 0xffff0000) == 0x38700000 ||
-            (ip->insn_bin & 0xffff0000) == 0x38710000))
+           ((ip->insn_bin & 0xfff00000) == 0x38600000 ||  /* amswap.w  rd, rk,
```
Comment (yetist) [inline gas/config/tc-loongarch.c:659]: Indent with 2 spaces

File: gas/config/tc-loongarch.c (hunk 2 above)
Comment (Calring) [reply gas/config/tc-loongarch.c:684]: Got it.Thanks a lot.

**Pull request merged**